pkgs/development/compilers/gcc/default.nix

```
@@ -321,7 +321,7 @@ pipe ((callFile ./common/builder.nix {}) ({
     inherit langC langCC langObjC langObjCpp langAda langFortran langGo langD version;
     isGNU = true;
     hardeningUnsupportedFlags =
-      optional (!atLeast11) "zerocallusedregs"
+      optional (!(atLeast11 && (targetPlatform.isx86_64 || targetPlatform.isAarch64 || targetPlatform.isSparc))) "zerocallusedregs"
```

`nixfmt` would format it this way:
```suggestion
      optional (
        !(atLeast11 && (targetPlatform.isx86_64 || targetPlatform.isAarch64 || targetPlatform.isSparc))
      ) "zerocallusedregs"
```
I think it makes sense to follow the formatting rules for new/changed code even when not the entire file is formatted that way yet.

---

Sure, let's do that.